runtime/compiler/control/MethodToBeCompiled.hpp

```diff
@@ -44,6 +45,7 @@ namespace TR { class CompilationInfoPerThreadBase; }
 class TR_OptimizationPlan;
 #if defined(J9VM_OPT_JITSERVER)
 namespace JITServer { class ServerStream; }
+class AOTCacheClassChainRecord;
```

Comment:
I think this declaration can be removed now

Reply:
True. Should I push this change now or wait until the tests are done?